stable/distributed-tensorflow/README.md

```diff
@@ -9,24 +9,24 @@ TensorFlow is an open source software library for numerical computation using da
 
 ## Prerequisites
 
-- Kubernetes cluster v1.8+ 
+- Kubernetes cluster v1.16+
```

Comment:
This verison is quite high, the given k8s api versions are available AFAIR since 1.10, so i sugest to safely write here 1.12 :)

```suggestion
- Kubernetes cluster v1.12+
```

Reply:
Done